### ISSUE TITLE
Can't find any field or property with name 'wrappedNode'.

### ISSUE THREAD

**Issue by javydreamercsw** [lukas-krecan/JsonUnit#157]
I have the following JSON object I'm trying to compare with but I'm getting a weird error I have no clue what it means. I assume this is some internal error since I don't have any node named wrappedNode.

Here's the actual Object I'm comparing (newUser):
```json{
  "_links" : {
    "self" : {
      "href" : "some URL"
    },
    "password" : {
      "href" : "some URL"
    },
    "password.set" : {
      "href" : "some URL"
    },
    "password.reset" : {
      "href" : "some URL"
    },
    "password.validate" : {
      "href" : "some URL"
    },
    "password.recover" : {
      "href" : "some URL"
    },
    "account.sendVerificationCode" : {
      "href" : "some URL"
    },
    "environment" : {
      "href" : "some URL"
    },
    "population" : {
      "href" : "some URL"
    }
  },
  "id" : "some id",
  "environment" : {
    "id" : "some id"
  },
  "population" : {
    "id" : "some id"
  },
  "createdAt" : "2019-01-09T19:59:00.490Z",
  "email" : "do-not-send@email.com",
  "enabled" : false,
  "lifecycle" : {
    "status" : "ACCOUNT_OK"
  },
  "mfaEnabled" : true,
  "name" : {
    "given" : "Test",
    "family" : "User",
    "formatted" : "Formatted Test User",
    "middle" : "POST",
    "honorificPrefix" : "Dr."
  },
  "updatedAt" : "2019-01-09T19:59:00.490Z",
  "username" : "username",
  "photo" : {
    "href" : "some URL"
  },
  "address" : {
    "streetAddress" : "1100 Ave",
    "locality" : "Somewhere",
    "region" : "TX",
    "postalCode" : "00000",
    "countryCode" : "US"
  }
}
```

And the object I'm comparing against (user):
```json{
  "population" : {
    "id" : "some Id",
    "createdAt" : "2019-01-08T20:31:23.186Z",
    "environment" : {
      "id" : "some Id"
    },
    "userCount" : 2,
    "_links" : {
      "self" : {
        "href" : "some URL"
      }
    },
    "name" : "Default",
    "description" : "Automatically created population",
    "updatedAt" : "2019-01-08T20:31:23.186Z"
  },
  "email" : "do-not-send@email.com",
  "enabled" : false,
  "mfaEnabled" : true,
  "name" : {
    "given" : "Test",
    "family" : "User",
    "formatted" : "Formatted Test User",
    "middle" : "POST",
    "honorificPrefix" : "Dr.",
    "honorificSufix" : "III"
  },
  "username" : "username",
  "photo" : {
    "href" : "some URL"
  },
  "address" : {
    "streetAddress" : "1100 Ave",
    "locality" : "Somewhere",
    "region" : "TX",
    "postalCode" : "00000",
    "countryCode" : "US"
  }
}
```

With this code:
```java
assertThatJson(
              newUser)
              .isObject()
              .isEqualToIgnoringGivenFields(user,
                      "_links", "userCount", "population", "environment", 
                      "createdAt", "lifecycle", "updatedAt");
```

**Comment by lukas-krecan** [comment]
Thanks for feedback, `isEqualToIgnoringGivenFields` is currently not supported. I have to figure out if it makes sense to support it since JSON document does not have properties nor fields. On the other hand, I see why you tried to use it. In the meantime I can only recommend you to use this https://github.com/lukas-krecan/JsonUnit#ignorevalues

**Comment by javydreamercsw** [comment]
That seems painful. Maybe the name needs changing but I thought it was exactly what I needed.

The link you provided was not very clear on how to use it. Maybe an example for my problem above would help.

**Comment by javydreamercsw** [comment]
If not supported it should throw a not supported exception.

**Comment by lukas-krecan** [comment]
Released as 2.3.0. It was simple to implement, so I did it. Other similar methods throw UnsupportedOperationException

**Comment by javydreamercsw** [comment]
Thanks a lot. Will try it tomorrow.
